test/datasource.test.js

```diff
@@ -80,6 +80,24 @@ describe('loopback:datasource generator', function() {
     });
   });
 
+  it('should support custom connector', function(done) {
+    var modelGen = givenDataSourceGenerator();
+    helpers.mockPrompt(modelGen, {
+      name: 'test-custom',
+      customConnector: 'loopback-component-storage',
```

Comment:
Please use a name that will never become a connector, e.g. `async` or `lodash`. I think the storage component is sort of connector-like.